lib/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -66,6 +67,24 @@ class ExploreScreen extends StatelessWidget {
                       ),
                     ),
                     const SizedBox(width: 20),
+                    const Expanded(
+                      child: SizedBox.shrink(),
+                    ),
```

Comment:
one line plz
 const Expanded(child: SizedBox.shrink()),

Reply:
Thanks (: , Will fix it on next pr to get it merged now.